toolchain/sem_ir/ids.cpp

```diff
@@ -90,19 +94,18 @@ auto NameId::ForIdentifier(IdentifierId id) -> NameId {
 }
 
 auto NameId::Print(llvm::raw_ostream& out) const -> void {
-  out << "name";
   if (*this == SelfValue) {
-    out << "SelfValue";
+    out << Label << "SelfValue";
```

Comment:
This is going to be camel case as well (`nameSelfValue`, `nameSelfType`, ...). Should it be changed too?

Reply:
Added parentheses.